jupyter-on-gke/storage.md

```diff
@@ -9,3 +9,40 @@ GCSFuse allow users to mount GCS Buckets as their local filesystem. This option
 ![Profiles Page](images/gcs_bucket.png)
 
 Since this bucket in GCS, there is built in permission control and access outside of the clutser.
+
+## Filestore
+
+Filestore has built in auto provisioning. This means that the first time this option is picked for a users, it will auto provision a Filestore instance then mount it to the Jupyter instance.
+
+**Important Note:** Currently the teir used is the `standard` tier. Different other teirs may require different handling.
```

Comment:
typo: tiers

Reply:
Also it is important to point to the filestore storageclass network param. If using a non default n/w for the GKE cluster, the user needs to point to the right network (https://cloud.google.com/kubernetes-engine/docs/how-to/persistent-volumes/filestore-csi-driver#storage-class) https://screenshot.googleplex.com/gdZujwkZ2Wu6MR5